packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -207,39 +334,20 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
             this.refreshModelChildren();
             return;
         }
+
+        // Searches in the dirty editors first.
+        this.searchInDirtyFiles(searchTerm);
```

Comment:
we should respect all search options for dirty file search as well

Reply:
Yes I just realized it. I am working on the support for search options right now ;)